<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>999</width>
    <height>833</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_img_dir">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Image Dir</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_img_dir">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_img">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>50</y>
      <width>640</width>
      <height>640</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_xmin">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>700</y>
      <width>56</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>xmin</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_ymin">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>720</y>
      <width>56</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>ymin</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_ymax">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>760</y>
      <width>56</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>ymax</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_xmax">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>740</y>
      <width>56</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>xmax</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_xmin_val">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>700</y>
      <width>56</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_ymin_val">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>720</y>
      <width>56</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_xmax_val">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>740</y>
      <width>56</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_ymax_val">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>760</y>
      <width>56</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_save">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>710</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>SAVE</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_labeling">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>710</y>
      <width>56</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Labeling</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_label">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>710</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget_img_list">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>311</width>
      <height>731</height>
     </rect>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>300</number>
    </attribute>
    <column>
     <property name="text">
      <string>이미지</string>
     </property>
    </column>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>999</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
